backend/models.js

```diff
@@ -0,0 +1,283 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+const TableContentSchema = new Schema({
+    agencyNumber: {
+        type: Number,
+        required: true
+    },
+    name: {
+        type: String,
+        required: true
+    },
+    status: {
+        type: String,
+        required: true
+    },
+    region: {
+        type: String,
+        required: true
+    },
+    city: {
+        type: String,
+        required: true
+    },
+    phone: {
+        type: String,
+        required: true
+    },
+    staff: {
+        type: String,
+        required: true
+    }
+});
+
+const ContactSchema = new Schema({
+    contact: {
+        type: String,
+        required: true
+    },
+    position: {
+        type: String,
+        required: true
+    },
+    phoneNumber: {
+        type: String
+    },
+    email: {
+        type: String,
+        required: true
+    }
+});
+
+const DistributionDaysSchema = new Schema({
+    monday: {
+        type: Boolean,
+        required: true
+    },
+    tuesday: {
+        type: Boolean,
+        required: true
+    },
+    wednesday: {
+        type: Boolean,
+        required: true
+    },
+    thursday: {
+        type: Boolean,
+        required: true
+    },
+    friday: {
+        type: Boolean,
+        required: true
+    },
+    saturday: {
+        type: Boolean,
+        required: true
+    },
+    sunday: {
+        type: Boolean,
+        required: true
+    }
+});
+
+const AgencySchema = new Schema({
+    tableContent: {
+        type: TableContentSchema,
```

Comment:
maybe add a brief comment here about how this can be used to access the data in the schema above